config/dev.exs

```diff
@@ -66,11 +66,11 @@ config :media, MediaWeb.Endpoint,
   ]
 
 config :ex_aws,
-  access_key_id: "YOUR_AWS_ACCESS_KEY_ID",
-  secret_access_key: "YOUR_AWS_ACCESS_KEY_SECRET"
+  access_key_id: "AKIAV3VNCU4LJWQ6W4CD",
+  secret_access_key: "a01xDHRxf2mgwo1NnnD89ygS03v8l3smqPrEVdP0"
```

Comment:
@zaher-hs this is a public github repo... these credentials can be seen publicly... you might want to update your secret key to avoid having people abusing your bucket...

Reply:
oh I pushed this by mistake will revert.